src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
+        public void Add(ushort jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(uint jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ulong jsonValue) { }
+        public void Clear() { }
+        public bool Contains(bool jsonValue) { throw null; }
+        public bool Contains(byte jsonValue) { throw null; }
+        public bool Contains(double jsonValue) { throw null; }
+        public bool Contains(short jsonValue) { throw null; }
+        public bool Contains(int jsonValue) { throw null; }
+        public bool Contains(long jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(sbyte jsonValue) { throw null; }
+        public bool Contains(float jsonValue) { throw null; }
+        public bool Contains(string jsonValue) { throw null; }
+        public bool Contains(System.Text.Json.JsonNode jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ushort jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(uint jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ulong jsonValue) { throw null; }
+        public void CopyTo(System.Text.Json.JsonNode[] array, int arrayIndex) { }
+        public System.Collections.Generic.IEnumerator<System.Text.Json.JsonNode> GetEnumerator() { throw null; }
+        public int IndexOf(System.Text.Json.JsonNode item) { throw null; }
+        public void Insert(int index, bool item) { }
+        public void Insert(int index, byte item) { }
+        public void Insert(int index, double item) { }
+        public void Insert(int index, short item) { }
+        public void Insert(int index, int item) { }
+        public void Insert(int index, long item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, sbyte item) { }
+        public void Insert(int index, float item) { }
+        public void Insert(int index, string item) { }
+        public void Insert(int index, System.Text.Json.JsonNode item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, ushort item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, uint item) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Insert(int index, ulong item) { }
+        public bool Remove(System.Text.Json.JsonNode item) { throw null; }
+        public void RemoveAt(int index) { }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class JsonBool : System.Text.Json.JsonNode
+    {
+        public JsonBool() { }
+        public JsonBool(bool value) { }
+        public bool GetBool() { throw null; }
+        public static implicit operator System.Text.Json.JsonBool (bool value) { throw null; }
```

Comment:
>value [](start = 72, length = 5)

Do we want to have a constant here similar to what we have in string.Empty? Does it even matter?